### ISSUE TITLE
Unwinding support

### ISSUE THREAD

**Issue by daxpedda** [hermit-os/kernel#172]
According to the discussion in https://github.com/hermitcore/uhyve/issues/30#issuecomment-792037267 this might still be impossible or very difficult at least.

It would make it much easier to host multiple "applications" in the same unikernel without a panic crashing all of it.

**Comment by stlankes** [comment]
Yes, I agree. Currently, I don't find the time to do it. But we are thinking about it.
